cherryontech/do-five-things
# [render preview] Rails portion of adding/updating tasks

Steps to test this pull request:
1. Run the migration locally:
```
bin/rails db:migrate RAILS_ENV=development
```
2. Kick off the servers:
```
bin/vite dev --clobber
bin/rails server -p 3002
```

_Note: I'm using port `3002` to help avoid conflicts with other projects._

3. Open the application in your browser:
-  Go to: `http://localhost:3002`
-  Enter 5 tasks and click Save.

4. Verify saved tasks:
- Refresh browser to confirm that the tasks have been saved

5. Check saved tasks in the console (alternative method):
- In a new terminal window:
```
bundle exec rails c
```
- From the rails console:
```
Task.all     # View all tasks
Task.today   # View today's tasks
```
